misc/scripts/install-local.sh

```diff
@@ -461,7 +461,10 @@ Pin-Priority: -1" | sudo tee /etc/apt/preferences.d/"${name}-pin" > /dev/null
 ask "Do you want to view/edit the pacscript" N
 if [[ $answer -eq 1 ]]; then
     if [[ -n $PACSTALL_EDITOR ]]; then
-        $PACSTALL_EDITOR "$PACKAGE".pacscript
+        $PACSTALL_EDITOR "$PACKAGE".pacscript || {
+            fancy_message warn "'$PACSTALL_EDITOR' not found, falling back to '$EDITOR'"
+            $EDITOR "$PACKAGE".pacscript
```

Comment:
```suggestion
            fancy_message warn "'$PACSTALL_EDITOR' not found, falling back to `sensible-editor`"
            sensible-editor "$PACKAGE".pacscript
```

Reply:
Wouldn’t back-bracing `sensible-editor` make it run inside the message?